docs/tinymce-per-block/src/assets/stylesheets/main.scss

```diff
@@ -1,19 +1,19 @@
-@import 'variables';
```

Comment:
I think we ought to exclude `docs` from these rules. They're more legacy prototypes than actively maintained.